src/api/__tests__/authorize.test.ts

```diff
@@ -77,14 +80,15 @@ describe('api', () => {
       const state = 'state';
       const country = 'JP';
       const scopes = 'points_read';
+      const samlSubjectId = 'mySubject'
 
       const mtLinkSdk = new MtLinkSdk();
-      mtLinkSdk.init(clientId);
+      mtLinkSdk.init(clientId, {samlSubjectId});
 
       authorize(mtLinkSdk.storedOptions, {
         state,
         redirectUri,
-        scopes
+        scopes,
```

Comment:
```suggestion
        scopes
```

Reply:
lol  In 2022 do we still need to review these styling issues, I expected some pre-commit linter to fix these 😅

Reply:
Then I expect I don't have to see it in the PR.